404.html

```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: 404
+---
+<div class="container-lg px-3">
+  <header class="py-6">
+    <div class="pb-0 pt-md-4 py-lg-6">
+      <div class="d-md-flex flex-md-items-center flex-md-row-reverse flex-md-justify-center gutter-md-spacious">
+        <div class="col-md-8 text-center text-md-left">
+          <h1 class="alt-h2 text-uppercase lh-condensed text-brand-blue">
+            <code style="font-size: 1.05em">404</code>: Page not found
+          </h1>
+          <h2 class="alt-h2 lh-condensed">
+            Something’s missing.
```

Comment:
The `’` character can get messed up in some browsers, can you please change it to the standard straight `'`?

Reply:
I changed it to an `&rsaquo;`.